src/app/server/BUILD.gn

```diff
@@ -15,12 +15,21 @@
 import("//build_overrides/chip.gni")
 import("${chip_root}/src/app/common_flags.gni")
 
+# Move this inside the GroupDataProvider once implemented Issue #11075
+declare_args() {
+  chip_enable_Test_Group = false
```

Comment:
Suggest keeping the all-lowercase convention

Reply:
done